ui/src/app/testing/fake-aggregation.service.ts

```diff
@@ -0,0 +1,97 @@
+import {AggregationResponse, JobStatus, TimeFrame} from "../shared/model/models";
+
+const aggregationResponse: AggregationResponse =
+  {
+   aggregations: [
+     {
+       entries: [
+         {
+           label: "owner1",
+           statusCounts: {
+             counts: [
+               {
+                 count: 2,
+                 status: JobStatus.Succeeded
+               },
+               {
+                 count: 1,
+                 status: JobStatus.Failed
+               }
+             ]
+           }
+         },
+         {
+           label: "owner2",
+           statusCounts: {
+             counts: [
+               {
+                 count: 4,
+                 status: JobStatus.Succeeded
+               },
+               {
+                 count: 6,
+                 status: JobStatus.Failed
+               }
+             ]
+           }
+         }
+       ],
+       key: "Owner"
+     },
+     {
+       entries: [
+         {
+           label: "user1",
+           statusCounts: {
+             counts: [
+               {
+                 count: 5,
+                 status: JobStatus.Succeeded
+               },
+               {
+                 count: 6,
+                 status: JobStatus.Failed
+               }
+             ]
+           }
+         },
+         {
+           label: "user2",
+           statusCounts: {
+             counts: [
+               {
+                 count: 8,
+                 status: JobStatus.Succeeded
+               },
+               {
+                 count: 11,
+                 status: JobStatus.Failed
+               }
+             ]
+           }
+         }
+       ],
+       key: "User"
+     },
+
+   ],
+   summary: {
+    counts: [
+      {
+        count: 10,
+        status: JobStatus.Succeeded
+      },
+      {
+        count: 3,
+        status: JobStatus.Failed
+      }
+    ]
+   }
+  };
+
+export class FakeAggregationService {
+  queryAggregations(timeFrame: TimeFrame, projectId: string): Promise<AggregationResponse> | AggregationResponse {
```

Comment:
Please return a `Promise.resolve(aggregationResponse)` instead. This is a better simulation of the actual code, which will not be returning a response synchronously.

Reply:
fixed.